### DISCUSSION TITLE
Jm123下載失敗

### DISCUSSION THREAD

**Discussion by 123hi123** [hect0x7/JMComic-Crawler-Python#270] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/1654e85b-e0a2-48a8-a204-11782c545aec)
打開來是空的
只有名稱 沒有內容

**Comment by hect0x7** [comment]
禁漫的域名都5xx报错了，估计是换图片域名了，等我更新吧

**Comment by hect0x7** [comment, marked as answer]
可以用了：https://github.com/hect0x7/JMComic-Crawler-Python/actions/runs/10439113995
重新fork下我的repo

**Comment by 123hi123** [comment]
等等 是fork上面的對吧

**Comment by 123hi123** [comment]
為什麼傳了兩次一樣的東西

**Reply by 123hi123** [reply]
話說作者 失敗的原因是因為不支援三位數的jm嗎

**Reply by hect0x7** [reply]
> 話說作者 失敗的原因是因為不支援三位數的jm嗎

怎么可能😂是线路问题

**Reply by 123hi123** [reply]
欸欸那你試試看125 改了之後123是可以了
